moveit_core/robot_model/src/joint_model.cpp

```diff
@@ -90,6 +90,31 @@ int JointModel::getLocalVariableIndex(const std::string& variable) const
   return it->second;
 }
 
+bool JointModel::harmonizePosition(double* values, const Bounds& other_bounds) const
+{
+  if (type_ != REVOLUTE)
+    return false;
+
+  bool modified = false;
+  if (*values < other_bounds[0].min_position_)
```

Comment:
what if the joint is continuous?

Reply:
Which behaviour would you expect for continuous joints?

Reply:
Continuous joints have their "bounds" between -Pi and Pi. Hence `harmonizePosition` will constrain their joint position to this range as well.